src/lib.rs

```diff
@@ -83,6 +83,8 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![cfg_attr(test, type_length_limit = "80000000")]
 
+#![doc(html_root_url = "https://docs.rs/mongodb/2.0.0-alpha.1")]
```

Comment:
seems like something we'd want to make sure we update programmatically with each release?

Reply:
this would probably be worth adding to the release checklist doc